packages/nimble-components/src/table/components/header/template.ts

```diff
@@ -1,9 +1,22 @@
-import { html } from '@microsoft/fast-element';
+import { html, when } from '@microsoft/fast-element';
+import { DesignSystem } from '@microsoft/fast-foundation';
 import type { TableHeader } from '.';
+import { IconArrowDown } from '../../../icons/arrow-down';
+import { IconArrowUp } from '../../../icons/arrow-up';
+import { TableColumnSortDirection } from '../../types';
 
 // prettier-ignore
 export const template = html<TableHeader>`
-    <template role="columnheader">
+    <template role="columnheader" aria-sort="${x => x.ariaSort}">
```

Comment:
A few thoughts/questions on accessibility after looking at the accessibility tree in Firefox and playing with VoiceOver:
1. I couldn't get `aria-sort` to appear in the accessibility tree or to be read by VoiceOver despite it definitely being present on the `nimble-table-header` element. I'm not sure if this is a gap in those assistive technologies or an issue with how we're adding the attribute. Maybe they can only recognize it on `<th>` elements or something? [Edit] I tried [this W3C accessible table example](https://w3c.github.io/aria-practices/examples/table/sortable-table.html) and still couldn't get the attribute to be mentioned so this may not be our problem.
2. In reading the [MDN docs for `aria-sort`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-sort) I saw they state that it should only be set once per table. In the case where a table is sorted by multiple columns should we only set the attribute on the column with the highest precedence?
3. The sort indicator icon is announced by voiceover but it has no information about what it is. Since it doesn't add information content beyond what's provided by the `aria-sort` attribute, what do you think about [marking it as decorative](https://www.w3.org/WAI/tutorials/images/decorative/)?

Reply:
I've added logic to make the `aria-sort` attribute only be configured on the column with the lowest `sort-index` (i.e., highest sort precedence). I also added `aria-hidden` on the sort indicator to hide it from the accessibility tree.